lib/eslint-angular/README.md

```diff
@@ -0,0 +1,42 @@
+# Alfresco ESlint Angular Library
```

Comment:
this is probably incorrect rebase, as it belongs to another PR

Reply:
rebased the branch - APSS-2135-replace-moment-to-date-fns now only my changes related to moment to date-fns conversion are showing.